packages/e2e-tests/src/tests/WrappedKeys.spec.ts

```diff
@@ -0,0 +1,1211 @@
+import { expect, jest } from '@jest/globals';
+import {
+  TinnyEnvironment,
+  getPkpSessionSigs,
+  getEoaSessionSigs,
+} from '@lit-protocol/tinny';
+import {
+  EthereumLitTransaction,
+  SerializedTransaction,
+  api,
+} from '@lit-protocol/wrapped-keys';
+import {
+  Connection,
+  Keypair,
+  LAMPORTS_PER_SOL,
+  PublicKey,
+  SystemProgram,
+  Transaction,
+  clusterApiUrl,
+} from '@solana/web3.js';
+
+import nacl from 'tweetnacl';
+import bs58 from 'bs58';
+import { ethers } from 'ethers';
+import {
+  AuthSig,
+  LIT_NETWORKS_KEYS,
+  SessionSigsMap,
+} from '@lit-protocol/types';
+import { encryptString } from '@lit-protocol/encryption';
+
+import { LIT_CHAINS } from '@lit-protocol/constants';
+
+// Using absolute pathing as these members are not exported from the module
+import { LIT_ACTION_CID_REPOSITORY } from '../../../wrapped-keys/src/lib/lit-actions-client/constants';
+import { getPkpAccessControlCondition } from '../../../wrapped-keys/src/lib/utils';
+import { LIT_PREFIX } from '../../../wrapped-keys/src/lib/constants';
+
+const {
+  importPrivateKey,
+  signTransactionWithEncryptedKey,
+  signMessageWithEncryptedKey,
+  generatePrivateKey,
+  exportPrivateKey,
+} = api;
+
+/**
+ * Helper migrated from tinny utils due to an issue with root level dependency resolve
+ * Migration is acceptible due to aggreagation of test casses into a snigle file context.
+ * @returns {string}
+ */
+export function randomSolanaPrivateKey() {
+  const BASE58_ALPHABET =
+    '123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz';
+  const SOLANA_PRIVATE_KEY_LENGTH = 88;
+
+  let result = '';
+  const charactersLength = BASE58_ALPHABET.length;
+  for (let i = 0; i < SOLANA_PRIVATE_KEY_LENGTH; i++) {
+    const randomIndex = Math.floor(Math.random() * charactersLength);
+    result += BASE58_ALPHABET.charAt(randomIndex);
+  }
+  return result;
+}
+
+try {
+  jest.setTimeout(100_0000);
+} catch (e) {
+  // ... continue execution
+}
+
+describe('Wrapped Keys', () => {
```

Comment:
Missing calls `releasePrivateKey` here.  Since _all_ of the tests in this suite _only_ use alice, this is probably easiest done by moving `alice` into the describe() block as a local mutable variable, and setting it and releasing the key in `beforeEach()` and `afterEach()`